src/actions/actionUnbindText.tsx

```diff
@@ -0,0 +1,36 @@
+import { getNonDeletedElements } from "../element";
+import { mutateElement } from "../element/mutateElement";
+import { getBoundTextElement } from "../element/textElement";
+import { ExcalidrawTextElement } from "../element/types";
+import { getSelectedElements } from "../scene";
+import { register } from "./register";
+
+export const actionUnbindText = register({
+  name: "unbindText",
+  contextItemLabel: "labels.unbindText",
+  perform: (elements, appState) => {
+    const selectedElements = getSelectedElements(
+      getNonDeletedElements(elements),
+      appState,
+    );
+    selectedElements.forEach((element) => {
+      const boundTextElement = getBoundTextElement(element);
+      if (boundTextElement) {
+        mutateElement(boundTextElement as ExcalidrawTextElement, {
+          containerId: null,
+          originalText: boundTextElement.text,
```

Comment:
this will persist the soft & hard line breaks — not sure that's what we wanna do

Reply:
what do you mean ? we want to make sure the text which was last edited is exactly same as original text once unbind

Reply:
Hmm ok I get it, you mean treplace with original text but with recomputed measurement

Reply:
done 👇🏻 
![Excalidraw (19)](https://user-images.githubusercontent.com/11256141/151985577-8dcdc9e4-f7c1-487b-9a02-1fb9dca54f51.gif)

Reply:
👍 btw, once we support #4267 we can keep the bounded dimensions

Reply:
yep